SPECS-EXTENDED/azl-otel-collector/azl-otel-collector.spec

```diff
@@ -0,0 +1,63 @@
+%global debug_package   %{nil}
```

Comment:
why this one?

Reply:
it's needed to avoid the error where no debug symbols need to be packaged. Added a comment used across various packages in azure linux